app/controllers/orders_controller.rb

```diff
@@ -1,6 +1,8 @@
 class OrdersController < EmployeeBaseController
   def index
     @orders = Order.where(employee: current_employee).includes(:reward)
+    @orders = @orders.delivered if order_params[:query] == 'delivered'
```

Comment:
I would name that params status - query is a bit too broad, and when you will add some more parameters for filtering or any other actions you would have to rename it anyways 

```suggestion
    @orders = @orders.delivered if order_params[:status] == 'delivered'
```